src/morphology.cpp

```diff
@@ -137,14 +137,26 @@ const Points& Morphology::points() const
 {
     return get<Property::Point>();
 }
+
+const std::vector<uint32_t> Morphology::sectionOffsets() const
+{
+    const std::vector<std::array<int, 2>>& indices_and_parents = get<Property::Section>();
```

Comment:
auto?

Reply:
Until the day code editors will replace `auto` by the real type in the visual display, I find its more confusing than anything else for the reader.